native-assets/NFT/README.md

```diff
@@ -0,0 +1,116 @@
+<p align="center">
+    <picture>
+        <source media="(prefers-color-scheme: dark)" srcset=".docs/nft-logo_white.png">
+        <img alt="light theme" src=".docs/nft-logo_black.png">
+    </picture>
+</p>
+
+<p align="center">
+    <a href="https://crates.io/crates/forc/0.46.1" alt="forc">
+        <img src="https://img.shields.io/badge/forc-v0.46.1-orange" />
+    </a>
+    <a href="https://crates.io/crates/fuel-core/0.20.4" alt="fuel-core">
+        <img src="https://img.shields.io/badge/fuel--core-v0.20.4-yellow" />
+    </a>
+</p>
+
+## Overview
+
+A non-fungible token (NFT) is a unique token that has a maximum supply of one. On the Fuel Network, all NFTs are [Native Assets](https://fuellabs.github.io/sway/v0.46.1/book/blockchain-development/native_assets.html). They are commonly associated with artwork / collectibles however there are many greater utilities beyond that which have yet to be written for the Fuel Network.
```

Comment:
```suggestion
A non-fungible token (NFT) is a unique token that has a maximum supply of one. On the Fuel Network, all NFTs are [Native Assets](https://docs.fuel.network/docs/sway/blockchain-development/native_assets). They are commonly associated with artwork / collectibles however there are many greater utilities beyond that which have yet to be written for the Fuel Network.
```

The link isn't 1-to-1 so check it out first. The docs portal is currently a minor version behind too. We're trying to push people towards the portal though.

Also, your link leads to a 404 for me